src/app/containers/Video/fixtureData.jsx

```diff
@@ -0,0 +1,187 @@
+                  blocks: [
+                    {
+                      model: {
+                        blocks: [
+                          {
+                            model: {
+                              attributes: [],
+                              text:
+                                'The residents of Ligsalz8 believe they have found the answer to the expensive cost of renting in Munich',
+                            },
+                            type: 'fragment',
+                          },
+                        ],
+                        text:
+                          'The residents of Ligsalz8 believe they have found the answer to the expensive cost of renting in Munich',
+                      },
+                      type: 'paragraph',
+                    },
+                  ],
+                },
+                type: 'text',
+              },
+            ],
+          },
+          type: 'caption',
+        },
+        {
+          model: {
+            blocks: [
+              {
+                model: {
+                  blocks: [
+                    {
+                      model: {
+                        blocks: [
+                          {
+                            model: {
+                              attributes: [],
+                              text: 'group of residents',
+                            },
+                            type: 'fragment',
+                          },
+                        ],
+                        text: 'group of residents',
+                      },
+                      type: 'paragraph',
+                    },
+                  ],
+                },
+                type: 'text',
+              },
+            ],
+          },
+          type: 'altText',
+        },
+      ],
+    },
+    type: 'image',
+  },
+  {
+    type: 'rawVideo',
+    model: {
+      guidance: null,
+      isLive: false,
+      duration: 'PT2H34M25S',
+      allowOffsiteEmbedding: false,
+      locator: 'urn:bbc:pips:pid:p064nsyw',
+    },
+  },
+  {
+    type: 'altText',
+    model: {
+      blocks: [
+        {
+          type: 'text',
+          model: {
+            blocks: [
+              {
+                type: 'paragraph',
+                model: {
+                  text:
+                    'This video shows Marks and Spencer CEO talking about the announcement earlier this morning',
+                  blocks: [
+                    {
+                      type: 'fragment',
+                      model: {
+                        text:
+                          'This video shows Marks and Spencer CEO talking about the announcement earlier this morning',
+                        attributes: [],
+                      },
+                    },
+                  ],
+                },
+              },
+            ],
+          },
+        },
+      ],
+    },
+  },
+];
+
+const videoBlockWithCaption = [...videoBlock];
+videoBlockWithCaption.push({
```

Comment:
Instead of pushing the caption block into the array here, you can define a caption block and adding it to the blocks array below, for each case `VideoWithCaption` & `VideoAmpWithCaption`.

Reply:
@sareh the `videoBlock` has more than one object in it, so I'm not sure how this would work? like it is okay to do a `videoblock.join(captionblock)` inline?

Reply:
I meant something like this:
```
export const VideoWithCaption = generateFixtureData({
  blocks: [videoBlock, captionBlock],
});
```